runtime/common/tests/xcm_mock/parachain.rs

```diff
@@ -209,8 +210,35 @@ pub type MultiAssetTransactor = MultiAssetAdapter<
     ParachainAssetConfig,
 >;
 
-pub type XcmRouter = super::ParachainXcmRouter<ParachainInfo>;
-pub type Barrier = AllowUnpaidExecutionFrom<Everything>;
+pub type XcmRouter = super::ParachainXcmRouter<MsgQueue>;
+
+match_type! {
+    pub type ParentLocation: impl Contains<MultiLocation> = {
+        MultiLocation { parents: 1, interior: Here }
+    };
+}
+match_type! {
+    pub type ParentOrSiblings: impl Contains<MultiLocation> = {
+        MultiLocation { parents: 1, interior: Here } |
+        MultiLocation { parents: 1, interior: X1(_) }
+    };
+}
+pub type Barrier = (
```

Comment:
could you type alias to a runtime instead of recreating the type here?
If the code is meant to be identical, there's no need to duplicate it, right?

Reply:
Yeah we could directly import and use either the Dolphin or Calamari configurations.
I'm going to log this and do it separately, because we can import a bunch of other configurations as well.